plugin/pkg/admission/podtolerationrestriction/admission.go

```diff
@@ -162,6 +164,21 @@ func (p *podTolerationsPlugin) Admit(a admission.Attributes) error {
 		}
 	}
 
+	if qoshelper.GetPodQOS(pod) != api.PodQOSBestEffort {
+		finalTolerations = tolerations.MergeTolerations(finalTolerations, []api.Toleration{
+			{
+				Key:      algorithm.TaintNodeMemoryPressure,
+				Operator: api.TolerationOpExists,
+				Effect:   api.TaintEffectNoSchedule,
+			},
+			{
+				Key:      algorithm.TaintNodeDiskPressure,
```

Comment:
at this time, cpu and memory are the only supported qos compute resources.

i think application of this toleration for disk pressure needs to be more nuanced.

/cc @vishh @jingxu97